crates/configuration/src/version5/introspection.sql

```diff
@@ -1515,8 +1537,13 @@ WITH
           USING (operator_name)
         WHERE
           op.is_infix = 't'
-		ORDER BY
-		  map.exposed_name, op.*
+          AND (
+            map.operator_type is null
+            OR map.operator_type = op.argument1_type
+            OR op.argument1_cast_chain LIKE ('% -> ' || map.operator_type_name)
```

Comment:
so using the operator name and the type, i can know that `>>=` on `inet` should be called `network_supeq` and should be an operator for the type `cidr` which has a valid cast from `cidr` to `inet`.